packet_handler_map.go

```diff
@@ -53,13 +54,48 @@ type packetHandlerMap struct {
 
 var _ packetHandlerManager = &packetHandlerMap{}
 
+func setReceiveBuffer(c net.PacketConn, logger utils.Logger) {
+	conn, ok := c.(interface{ SetReadBuffer(int) error })
+	if !ok {
+		logger.Debugf("Connection doesn't allow setting of receive buffer size")
+		return
+	}
+	size, err := inspectReadBuffer(c)
+	if err != nil {
+		log.Printf("Failed to determine receive buffer size: %s", err)
```

Comment:
I'm using `log` here, so that users will actually see the log message (even if QUIC_GO_LOG_LEVEL is not set).

Reply:
Why would this always need to log? What are the failure modes here that require user action?

We support [custom pconns](https://godoc.org/github.com/lucas-clemente/quic-go#DialContext), which don't need to implement `SyscallConn()`, and could benignly trigger this?

Reply:
We do, but I'm assuming that all pconns at some point wrap a UDP conn. QUIC v1 is only defined to work over UDP anyway.